src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_MegaIndustrialApiary.java

```diff
@@ -1077,7 +1077,7 @@ public void updateTVar(float t) {
             }
 
             public void evaluate() {
-                this.amount = Bee.getFinalChance(chance, beeSpeed, 2.f, t);
+                this.amount = Bee.getFinalChance(chance, beeSpeed, 17.19926784f, t);
```

Comment:
no magic numbers pls. At least explains from where it comes in the comment, so monthes later we can still understand what is going on

Reply:
4*1.2^8